src/ontology/mondo-edit.obo

```diff
@@ -248858,14 +248860,15 @@ xref: ICD9:270.7 {source="MONDO:relatedTo", source="MONDO:i2s"}
 xref: MESH:D020158 {source="DOID:9268"}
 xref: NCIT:C84937 {source="MONDO:equivalentTo", source="DOID:9268"}
 xref: NORD:1512 {source="MONDO:NORD"}
-xref: OMIM:605899 {source="MONDO:equivalentTo", source="Orphanet:407", source="Orphanet:407/e", source="DOID:9268"}
+xref: OMIMPS:605899 {source="MONDO:equivalentTo"}
```

Comment:
I don't know that it is worthwhile to review every instance here- these probably need the same action as above